pkgs/development/python-modules/marisa-python/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonPackage, fetchgit, marisa, swig }:
+
+buildPythonPackage rec {
+  pname = "marisa-python";
+  version = "1.3.40";
+  name = "${pname}-${version}";
+
+  src = fetchgit {
```

Comment:
Better use `fetchFromGitHub`. This will download a tarball instead of cloning the repo.

Reply:
fixed